src/iframe-entry/pages/SignSetAssetScript/SignSetAssetScriptComponent.tsx

```diff
@@ -0,0 +1,260 @@
+                                            variant="body2"
+                                            color="standard.$0"
+                                            isTruncated={true}
+                                        >
+                                            {assetId}
+                                        </Text>
+                                    </Copy>
+                                </Flex>
+                                <Flex mt="$20" alignItems="center">
+                                    <Text
+                                        variant="body2"
+                                        color="basic.$500"
+                                        mr="3px"
+                                    >
+                                        Smart Asset Script
+                                    </Text>
+                                    <Help
+                                        showDelay={1500}
+                                        popperOptions={popperOptions}
+                                    >
+                                        <Text
+                                            display="block"
+                                            variant="body1"
+                                            fontWeight={700}
+                                        >
+                                            A Smart Asset is an asset with an
+                                            attached script that places
+                                            conditions on every transaction made
+                                            for the token in question.
+                                        </Text>
+                                        <Text
+                                            display="block"
+                                            variant="body2"
+                                            mt="5px"
+                                        >
+                                            Each validation of a transaction by
+                                            a Smart Asset's script increases the
+                                            transaction fee by 0.004 WAVES. For
+                                            example, if a regular tx is made for
+                                            a Smart Asset, the cost is 0.001 +
+                                            0.004 = 0.005 WAVES. If an exchange
+                                            transaction is made, the cost is
+                                            0.003 + 0.004 = 0.007 WAVES.
+                                            <ExternalLink
+                                                mt="$5"
+                                                variant="body2"
+                                                display="block"
+                                                href="https://docs.wavesplatform.com/en/building-apps/smart-contracts/smart-assets"
```

Comment:
External link has no target blank attribute by default?

Reply:
no it doesn't